packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter_driver/src/health.dart';
 import 'package:flutter_driver/src/message.dart';
 import 'package:json_rpc_2/json_rpc_2.dart' as rpc;
 import 'package:mockito/mockito.dart';
+import 'package:quiver/testing/async.dart';
```

Comment:
Done. Also did a general clean-up of `pubspec.yaml` (removed obsolete dep on `logging`, sorted alphabetically).